src/Approximations/overapproximate.jl

```diff
@@ -516,3 +516,180 @@ function overapproximate(cap::Intersection{N,
     return overapproximate(swap(cap), dir; kwargs...)
 end
 
+
+function load_taylormodels_overapproximation()  # function to be loaded by Requires
+return quote
+"""
+         overapproximate(vTM::Vector{TaylorModel1{T, S}},
+                            ::Type{Zonotope})where{T, S}
+
+Convert between TaylorModel1 and in Zonotope.
+
+### Input
+
+
+- `vTM` -- Vector of TaylorModel1
```

Comment:
```suggestion
- `vTM` -- vector of `TaylorModel1`
```

(in general note that code has to be written inside backticks so that it renders as such in the REPL)